wormhole-connect/src/utils/transferValidation.ts

```diff
@@ -311,7 +317,7 @@ export const validateAll = async (
     receiveAmount: '',
   };
 
-  if (isAutomatic) {
```

Comment:
same comment here - this is generic code but now has NTT-specific logic

Reply:
Agree, routes should be able to validate themselves